addOns/websocket/src/main/javahelp/org/zaproxy/zap/extension/websocket/resources/help/contents/pscanrules.html

```diff
@@ -11,5 +11,22 @@ <H1>WebSocket Passive Scan Rules</H1>
         <h2 id="scripts">Scripts</h2>
         Scripts which are included by default in the add-on and they implement the following WebSocket passive scan rules:
 
+        <h2 id="ip_add_disc">Private Address Disclosure</h2>
+        Checks incoming WebSocket message payload for inclusion of RFC 1918 IPv4 addresses as well as Amazon EC2 private hostnames (for example, ip-10-0-56-78). This information can give an attacker useful information about the IP address scheme of the internal network, and might be helpful for further attacks targeting internal systems. <br>
+
+        This passive scanner may generate false positives in the case of larger dotted numeric strings, such as vp09.02.51.10.01.09.16, where the latter 4 octets appear to be a RFC 1918 IPv4 address. After review an analyst can mark such alerts as False Positives in ZAP.<br>
+        <br>
+        <table border="1"  width = "200">
+            <tr><th>Use case</th><th>Output</th></tr>
+            <tr><td>10.255.255.255:</td><td>True Positive</td></tr>
```

Comment:
Should the trailing colon be included?

Reply:
There wasn't necessary, so it is removed